src/test/java/training/article/getArticle.feature

```diff
@@ -0,0 +1,26 @@
+Feature:  get an article
+
+  Background:
+    * url url
+    * def responseGetArticle = read('training/article/responsesuccessfully.json')
+
+  @getArticle
+  Scenario:  get an article
+
+    * def getArticles = call read('getArticle.feature@getAllArticles')
+    * def randomArticle = read('../helpers/Random.js')
+    * def slug = typeof passedSlug == 'undefined' ? randomArticle(getArticles.response.articles).slug : passedSlug
```

Comment:
you don't need to do this. Remember that you always called this scenario with a specific article.